app/bundles/CoreBundle/IpLookup/IP2LocationBinLookupDB4.php

```diff
@@ -0,0 +1,82 @@
+<?php
+
+/*
+ * @copyright   2017 Mautic Contributors. All rights reserved
+ * @author      Mautic
+ *
+ * @link        http://mautic.org
+ *
+ * @license     GNU/GPLv3 http://www.gnu.org/licenses/gpl-3.0.html
+ */
+
+namespace Beechit\IpBundle\IpLookup;
+
+namespace Mautic\CoreBundle\IpLookup;
```

Comment:
I've never seen multiple namespaces defined for one class. What's the reason for it?